### PR TITLE
Setup libarary

### PR DESCRIPTION
**Pull request by svenvanderburg**
This PR is just for reviewing purposes!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # pa-algorithms
-Repository with algorithms for Physical Activity.
\ No newline at end of file
+Repository with Perfect Fit algorithms for physical activities and training plans.
```
Comment (svenvanderburg) [inline README.md:2]: Could you add some installation instructions? Also see https://stackoverflow.com/questions/20101834/pip-install-from-git-repo-branch.

**Review by svenvanderburg — COMMENTED** [review]
Looks good @bscheltinga ! I just left a small comment.

File: README.md (hunk 1 above)
Comment (bscheltinga) [reply README.md:2]: Hi @svenvanderburg, I added a line with installation instructions.

Comment (svenvanderburg) [reply README.md:2]: Cool! Thanks :)

**Comment by svenvanderburg** [comment]
Looks good now!

**Pull request closed without merging**